ggml.h

```diff
@@ -244,11 +244,10 @@
 #define GGML_ASSERT(x) \
     do { \
         if (!(x)) { \
-            fprintf(stderr, "GGML_ASSERT: %s:%d: %s\n", __FILE__, __LINE__, #x); \
-            fflush(stderr); \
```

Comment:
Is there any reason for not flushing `stderr`?

Reply:
The reason for removing this is described in the OP. What do you expect flushing stderr to do? We aren't modifying the buffering of stderr in this program, it is either unbuffered as is convention on Unix (always prints immediately) or line buffered (prints immediately after `\n`). If you can think of a legitimate use case of llama.cpp where setbuf is called to make stderr fully buffered, I would be fine with leaving the fflush.